app/Page_Builder/Widget/Employee_Layout.php

```diff
@@ -2,24 +2,24 @@
 namespace Boilerplate\Theme\Page_Builder\Widget;
 
 /**
- * Class Hero_Layout
+ * Class Employee_Layout
```

Comment:
you don't need to change anything with PB Layouts in your tasks. Why you do so?

Reply:
Returned back